src/moepkg/exmode.nim

```diff
@@ -1091,6 +1098,26 @@ proc shellCommand(status: var EditorStatus, shellCommand: string) =
 
   status.changeMode(currentBufStatus.prevMode)
 
+proc backgroundCommand(status: var EditorStatus) =
+  saveCurrentTerminalModes()
+  exitUi()
+  discard execShellCmd("printf \"Press Enter\"")
+  discard execShellCmd("read _")
+  restoreTerminalModes()
+  status.commandLine.clear
+  status.changeMode(currentBufStatus.prevMode)
+
+proc manualCommand(status: var EditorStatus, manualInvocationCommand: string) =
+  saveCurrentTerminalModes()
+  exitUi()
+
+  # TODO:  Configure a default manual page to show on `:man`.
+  discard execShellCmd(manualInvocationCommand)
```

Comment:
If it fails, I think it's better to display an error message.

```suggestion
  let exitCode = execShellCmd(manualInvocationCommand)
  restoreTerminalModes()

  if exitCode != 0:
    let mess = "Error: No manual entry for " & manualInvocationCommand
    status.commandLine.writeMessageOnCommandLine(mess, EditorColorPair.errorMessage)
    status.messageLog.add(mess.toRunes)
  else:
    status.commandLine.clear
```

Reply:
@fox0430

I applied the requested changes such that the error message will now be shown on the editor's command line, as well. :wrench: :heavy_check_mark: